src/lib/server/result_serializer.hpp

```diff
@@ -7,6 +7,8 @@
 
 namespace opossum {
 
+struct ExecutionInformation;
```

Comment:
Yes, I think an include would to the job as well. We could change it, but I will leave it, since (at least I think so) Forward Declarations instead of includes in Header Files can reduce compile time.

Reply:
Yepp. Whenever* possible, forward declarations are better than including the related header.

*) I feel like I need to add some relativization. There are probably cases where using the include helps with optimizations or something else. As a rule of thumb, forward declarations are better.